gpio/__init__.py

```diff
@@ -79,23 +74,30 @@ def parse_item(self, item):
             GPIO.setup(in_pin, GPIO.IN, pull_up_down=GPIO.PUD_UP)
             self.logger.debug("GPIO: INPUT {0} assigned to pin \'{1}\'".format(item, in_pin))
             self._items.append(item)
+            self._itemsdict[in_pin] = item
             return self.update_item      
         if self.has_iattr(item.conf, 'gpio_out'):
             out_pin = int(self.get_iattr_value(item.conf, 'gpio_out'))
+            GPIO.setup(out_pin, GPIO.IN, pull_up_down=GPIO.PUD_UP)
```

Comment:
The GPIO.IN is necessary to detect value changes in the output pin. Something like a "reply" whether it worked or not